packages/build/src/plugins_core/main.js

```diff
@@ -69,6 +73,14 @@ const isEdgeHandlersPlugin = function({ package }) {
 
 const EDGE_HANDLERS_LOCATION = 'edge-handlers'
 
+const getDeployPlugin = function(BUILDBOT_SERVER_SOCKET) {
+  if (BUILDBOT_SERVER_SOCKET === undefined) {
```

Comment:
Instead of checking for `BUILDBOT_SERVER_SOCKET` (i.e. `--buildbotServerSocket` CLI flag), should we check for the `--triggerDeployWithBuildbotServer` CLI flag instead?

This might be useful if we were to use the buildbot TCP socket for other purposes than deploying?